awx/api/serializers.py

```diff
@@ -2825,6 +2826,25 @@ def get_summary_fields(self, obj):
         d['recent_jobs'] = self._recent_jobs(obj)
         return d
 
+    def validate(self, attrs):
+        webhook_service = attrs.get('webhook_service', getattr(self.instance, 'webhook_service', None))
+        webhook_credential = attrs.get('webhook_credential', getattr(self.instance, 'webhook_credential', None))
+
+        if webhook_credential and webhook_credential.credential_type.kind != 'token':
```

Comment:
`kind` has been the target of deprecation attempts

I would prefer `webhook_credential.credential_type.namespace`

Reply:
All right, how about now?